README.md

```diff
@@ -30,6 +30,7 @@ Run the command:
 python run.py [options]
 
 options:
+  -c CONFIG_PATH, --config_path CONFIG_PATH                                                                                                             specify config file for launch configuration
```

Comment:
it should be -c and --config-path ... there is no use of underscore in argument names